src/views/rebenchdb.R

```diff
@@ -131,9 +134,55 @@ get_profile_availability <- function(rebenchdb, hash_1, hash_2) {
   dbClearResult(qry)
   
   factorize_result(result)
+}
+  convert_array <- function(x) {
+    x <- gsub("(^\\{|\\}$)", "", x)
+    strsplit(x, split = ",")
+  }
+
+  convert_double_array <- function(x) {
+      sapply(convert_array(x),as.double)
+  }
+
+restore_iteration <- function(result) {
+    result <-
+    result |>     
+      mutate(value = convert_double_array(result$value)) 
+
+  print(result$value)
+  iterations <- list()
+  for(i in result$value)
+  {
+    y <- 1
+    for(j in i)
+    {      
+      iterations[[length(iterations) + 1]] <- y
+      y <- y+1
+    }
+  }
+
+  result <- unnest(result, cols = c(value)) 
+  
+  result <-
+    result |>     
+      mutate(iteration = as.integer(iterations)) 
+
+  result
+}
+
+cast_unnest_array <- function(result) {
+result <-
+   result %>% 
+    collect() %>%
```

Comment:
Didn't you say `collect()` isn't needed?
Please also use `|>` consistently instead of `%>%`.